packages/react/src/ComparisonTable/ComparisonTable.module.css

```diff
@@ -3,12 +3,17 @@
 */
 .ComparisonTable--heading {
   text-align: left;
+  max-width: 18ch;
+  margin-bottom: var(--brand-ComparisonTable-spacing-inner-heading);
```

Comment:
I think heading could be brought forward a little, as it's part of the pattern and could help leave door open to co-locate additional tokens.

WDYT about `--brand-ComparisonTable-heading-marginBottom`

☝️ This feels more targeted. @lukasoppermann just confirmed that marginBottom is acceptable too, which is neat.

Reply:
Sounds good, I've made the changes here: https://github.com/primer/brand/pull/313/commits/2f87494bbbcdc2ff4de6c9e5cc96db15cfc38111